OpenAI-Extension.sln

```diff
@@ -79,64 +79,216 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Functions.Worker.Extensions
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Functions.Worker.Extensions.OpenAI.AzureAISearch", "src\Functions.Worker.Extensions.OpenAI.AzureAISearch\Functions.Worker.Extensions.OpenAI.AzureAISearch.csproj", "{CAB7034D-FBA9-4A7D-B50B-E9029DFD0DEF}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "WebJobs.Extensions.OpenAI.CosmosDBSearch", "src\WebJobs.Extensions.OpenAI.CosmosDBSearch\WebJobs.Extensions.OpenAI.CosmosDBSearch.csproj", "{3B232EA0-9CAE-4488-8579-E86F1FCFEE9A}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Functions.Worker.Extensions.OpenAI.CosmosDBSearch", "src\Functions.Worker.Extensions.OpenAI.CosmosDBSearch\Functions.Worker.Extensions.OpenAI.CosmosDBSearch.csproj", "{FD53AA42-6DC5-422F-919C-FB33591DC9E7}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "rag-comosdb", "rag-comosdb", "{24193510-C974-42E2-AE1F-1097E16A2445}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SemanticCosmosDBSearchEmbeddings", "samples\rag-cosmosdb\SemanticCosmosDBSearchEmbeddings\SemanticCosmosDBSearchEmbeddings.csproj", "{D4EC72DC-5B7C-4C24-AA72-A18EF04041BC}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
+		Debug|arm64 = Debug|arm64
```

Comment:
wondering why arm64 and x86 configs are listed separately?

Reply:
I added it by accident. Removed now